examples/cis3-nft-sponsored-txs/Cargo.toml

```diff
@@ -14,6 +14,10 @@ crypto-primitives = ["concordium-std/crypto-primitives"]
 [dependencies]
 concordium-std = {path = "../../concordium-std", default-features = false}
 concordium-cis2 = {path = "../../concordium-cis2", default-features = false}
+ed25519-dalek = "1.0.1"
```

Comment:
I don't think we should need this. If we need it for integration testing then it should go under dev-dependencies I think.